appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.3.{build}-{branch}
+
+image: Visual Studio 2015
+
+cache:
+    - C:\MumbleBuild\cache -> appveyor.yml
+
+clone_folder: C:\MumbleBuild\mumble
+
+environment:
+    MUMBLE_BUILDSCRIPT: 1.3.x/mumble-win64-static.cmd
+    MUMBLE_ENVIRONMENT_VERSION: win64-static-no-ltcg-1.3.x-2017-02-02-ec94ddb-790
```

Comment:
Hmmm. I would have liked we would just use the latest available buildenv. But I guess that one is on me. :-)